csrc/scheduler/matmul_utils.cpp

```diff
@@ -361,6 +361,13 @@ std::shared_ptr<MatmulParams> getMatmulHeuristics(
   (void)runtime_info;
   auto params = std::make_shared<MatmulParams>();
 
+  // Set kernel index mode
+  params->cparams.index_type = runtime_info.getIndexType();
```

Comment:
`index_type` is needed in `compileFusion` and needs to be set before returning. This will avoid errors on architectures not supported by matmul scheduler (see `getMmaOp` function.)